### PR TITLE
Updating instructions to include Azure Stack Hub

### PR DESCRIPTION
**Pull request by garvitarai**
Added sample work flow for ASH and service principle instructions

@N-Usha please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,31 @@ jobs:
 
 Refer to the [Azure PowerShell](https://github.com/azure/powershell) Github action to run your Azure PowerShell scripts.
```
Comment (N-Usha) [inline README.md:77]: Before any of these changes are merged, we need a mention about the new "environment" parameter being introduced into the action.yml which supports all the new sovereign clouds and AzureStack.

File: README.md [hunk 2]
```diff
@@ -85,6 +110,13 @@ To function correctly, this service principal must be assigned the [Contributor]
 The following steps describe how to create the service principal, assign the role, and create a secret in your repository with the resulting credentials.
 
 1. Open the Azure Cloud Shell at [https://shell.azure.com](https://shell.azure.com). You can alternately use the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) if you've installed it locally. (For more information on Cloud Shell, see the [Cloud Shell Overview](https://docs.microsoft.com/azure/cloud-shell/overview).)
+
+    1.1 **(Required for Azure Stack Hub)** Run the following command to set the SQL Management endpoint to 'not supported'
```
Comment (N-Usha) [inline README.md:114]: Please change this to "Required ONLY when **environment** is Azure Stack Hub" coz I assume this step is optional when its not Azure Stack.

**Review by N-Usha — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (garvitarai) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (garvitarai) [reply README.md:77]: Added an introduction for environments parameter in the description for the Action. I am not too sure which sovereign clouds or what other capabilities you are supporting, so please updated based on context or let me know what to change

**Review by N-Usha — APPROVED** [review]

**Pull request merged**